README.md

```diff
@@ -1,6 +1,17 @@
 # Jade Explorer
 
-A block explorer for Ethereum based blockchains. The explorer uses Jade Service Runner for managing background services (Multi-Geth), OpenRPC for underlying functionality, and Pristine.
+A block explorer for Ethereum based blockchains. 
```

Comment:
What do you think about EVM-based instead of Ethereum based? EVM-based reference proceeds this paragraph as well.